src/framework/core/js/Fluid.js

```diff
@@ -224,6 +223,31 @@ var fluid = fluid || fluid_3_0_0;
         }
     };
 
+    fluid.notrycatch = false;
+
+    // A wrapper for the try/catch/finally language feature, to aid debugging in the QUnit UI by means of exception breakpoints for
+    // uncaught exceptions, since so many libraries, e.g. jQuery throw junk caught exceptions on startup
+    fluid.tryCatch = function (tryfun, catchfun, finallyfun) {
```

Comment:
Probably worthwhile to add some tests for `fluid.tryCatch`

Reply:
added